### PR TITLE
Fix Annotations

### LINKED ISSUE #11: Missing Type Annotations
There are missing type annotations for some functions in this library.

The `typing` module does not exist on CircuitPython devices so the import needs to be wrapped in try/except to catch the error for missing import. There is an example of how that is done here:

```python
try:
    from typing import List, Tuple
except ImportError:
    pass
```

Once imported the typing annotations for the argument type(s), and return type(s) can be added to the function signature. Here is an example of a function that has had this done already:

```python
def wrap_text_to_pixels(
    string: str, max_width: int, font=None, indent0: str = "", indent1: str = ""
) -> List[str]:
```
If you are new to Git or Github we have a guide about contributing to our projects here: https://learn.adafruit.com/contribute-to-circuitpython-with-git-and-github

There is also a guide that covers our CI utilities and how to run them locally to ensure they will pass in Github Actions here: https://learn.adafruit.com/creating-and-sharing-a-circuitpython-library/check-your-code In particular the pages: `Sharing docs on ReadTheDocs` and `Check your code with pre-commit` contain the tools to install and commands to run locally to run the checks.

If you are attempting to resolve this issue and need help, you can post a comment on this issue and tag both @foamyguy and @kattni or reach out to us on Discord: https://adafru.it/discord in the `#circuitpython-dev` channel.

The following locations are reported by mypy to be missing type annotations:

- [ ] adafruit_sgp40/voc_algorithm.py:93
- [ ] adafruit_sgp40/voc_algorithm.py:99
- [ ] adafruit_sgp40/voc_algorithm.py:102
- [ ] adafruit_sgp40/voc_algorithm.py:105
- [ ] adafruit_sgp40/voc_algorithm.py:137
- [ ] adafruit_sgp40/voc_algorithm.py:178
- [ ] adafruit_sgp40/voc_algorithm.py:206
- [ ] adafruit_sgp40/voc_algorithm.py:273
- [ ] adafruit_sgp40/voc_algorithm.py:278
- [ ] adafruit_sgp40/voc_algorithm.py:287
- [ ] adafruit_sgp40/voc_algorithm.py:302
- [ ] adafruit_sgp40/voc_algorithm.py:345
- [ ] adafruit_sgp40/voc_algorithm.py:391
- [ ] adafruit_sgp40/voc_algorithm.py:408
- [ ] adafruit_sgp40/voc_algorithm.py:559
- [ ] adafruit_sgp40/voc_algorithm.py:658
- [ ] adafruit_sgp40/voc_algorithm.py:665
- [ ] adafruit_sgp40/voc_algorithm.py:683
- [ ] adafruit_sgp40/voc_algorithm.py:687
- [ ] adafruit_sgp40/voc_algorithm.py:706
- [ ] adafruit_sgp40/voc_algorithm.py:709
- [ ] adafruit_sgp40/voc_algorithm.py:771
- [ ] adafruit_sgp40/\_\_init\_\_.py:110
- [ ] adafruit_sgp40/\_\_init\_\_.py:159
- [ ] adafruit_sgp40/\_\_init\_\_.py:178
- [ ] adafruit_sgp40/\_\_init\_\_.py:205
- [ ] adafruit_sgp40/\_\_init\_\_.py:228
- [ ] adafruit_sgp40/\_\_init\_\_.py:256
- [ ] adafruit_sgp40/\_\_init\_\_.py:294
- [ ] adafruit_sgp40/\_\_init\_\_.py:302

### PR DESCRIPTION
**Pull request by tcfranks**
resolves #11 
submitted for review / comment

### CODE REVIEW

**Review by tekktrik — CHANGES_REQUESTED** [review]
This one's a little bit more complex in the `voc_algorithm.py` file, so here's feedback for part 1 while I look at that next.

File: adafruit_sgp40/__init__.py [hunk 1]
```diff
@@ -225,7 +232,9 @@ def measure_raw(self, temperature=25, relative_humidity=50):
         self._measure_command = bytearray(_cmd)
         return self.raw
 
-    def measure_index(self, temperature=25, relative_humidity=50):
+    def measure_index(
+        self, temperature: float = 25, relative_humidity: float = 50
```
[the remark this answers is not in the document]
Comment (tcfranks) [reply adafruit_sgp40/__init__.py:236]: @tekktrik I don't understand.  Isn't that what I put in the updae?

[the remark this answers is not in the document]
Comment (tekktrik) [reply adafruit_sgp40/__init__.py:236]: Weird, it is, I guess I was looking at the wrong thing?  In any case, resolved!

**Comment by tcfranks** [comment]
resubmitted __init__.py with suggested change

**Review by tekktrik — APPROVED** [review]
Thanks!

**Pull request merged**